pkg/application/inject/fuse/injector.go

```diff
@@ -289,3 +294,36 @@ func (s *Injector) labelInjectionDone(pod common.FluidObject) error {
 func (s *Injector) getServerlessPlatformFromMeta(metaObj metav1.ObjectMeta) string {
 	return utils.GetServerlessPlatfrom(metaObj.Labels)
 }
+
+// check whether the podSpecs is using vineyard runtime
+func (s *Injector) checkVineyardPlatform(podSpecs *mutator.MutatingPodSpecs, runtimeInfos map[string]base.RuntimeInfoInterface) string {
```

Comment:
I think Vineyard is a runtime not platform.

Reply:
Thanks for this catch.